openshift/template.yaml

```diff
@@ -107,72 +79,72 @@ objects:
     triggers:
     - type: ConfigChange
     replicas: ${{REPLICAS}}
+- apiVersion: route.openshift.io/v1
+  kind: Route
+  metadata:
+    name: gremlin-api
+    labels:
+      type: oca-route
+      template: oca-gremlin-server
+  spec:
+    to:
+      kind: Service
+      name: oca-gremlin-${CHANNELIZER}
+      weight: 100
+    port:
+      targetPort: '8182'
 
 parameters:
-- description: Value of the 'storage.dynamodb.client.credentials.class-name' property in dynamodb.properties
-  name: DYNAMODB_CLIENT_CREDENTIALS_CLASS_NAME
-  required: true
-  value: "com.amazonaws.auth.DefaultAWSCredentialsProviderChain"
-- description: Value of the 'storage.dynamodb.client.endpoint' property in dynamodb.properties
-  name: DYNAMODB_CLIENT_ENDPOINT
-  required: false
-  value: "https://dynamodb.us-east-1.amazonaws.com"
-- description: Change the channelizer being used to deploy the gremlin server. Set this to "1" for HTTP channelizer, and "0" for WebSocket channelizer. Defaults to "0" i.e. WebSocket.
-  name: REST_VALUE
-  value: "0"
 - description: MEMORY_LIMIT
   name: MEMORY_LIMIT
   required: true
-  value: "1536Mi"
+  value: "2048Mi"
 - description: MEMORY_REQUEST
   name: MEMORY_REQUEST
   required: true
-  value: "1536Mi"
+  value: "1280Mi"
 - description: CPU_LIMIT
   name: CPU_LIMIT
   required: true
   value: "1000m"
 - description: CPU_REQUEST
   name: CPU_REQUEST
   required: true
-  value: "500m"
-- description: Name of the channelizer being used. Setting this will not change the channelizer, this will merely influence the names of OpenShift objects being created. To change the channelizer, see REST_VALUE.
-  name: CHANNELIZER
-  required: true
-  value: ws
-
+  value: "300m"
+  # TODO - Right now using docker.io for image registry, once CI/CD setup complete then we can move this to quay/internal registry
 - description: Docker registry where the image is
   displayName: Docker registry
   required: true
   name: DOCKER_REGISTRY
-  value: "quay.io"
-
+  value: "docker.io"
 - description: Docker image to use
   displayName: Docker image
   required: true
   name: DOCKER_IMAGE
-  value: "openshiftio/bayesian-gremlin"
-
+  value: "rajusem/gremlin-http"
 - description: Image tag
   displayName: Image tag
   required: true
   name: IMAGE_TAG
-  value: "1.1.0"
-
+  value: "cassandra"
 - description: Number of deployment replicas
   displayName: Number of deployment replicas
   required: true
   name: REPLICAS
   value: "1"
-
```

Comment:
Leave QUERY_ADMINISTRATION_REGION as it is. It will be helpful run multiple deployment of gremlin. e.g. one for online flow and another for offline flow.

Reply:
Added the same.